assets/common/src/log_utils/config.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Logger Config."""
+
+
+class AppName:
+    """Component AppName."""
+
+    IMPORT_MODEL = "import_model"
+    REGISTER_MODEL = "register_model"
+    DEPLOY_MODEL = "deploy_model"
+    MLFLOW_MODEL_LOCAL_VALIDATION = "mlflow_model_local_validation"
```

Comment:
shouldnt we have convert to mlflow component here

Reply:
That is under training folder , not common.